clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_utils.py

```diff
@@ -308,3 +308,18 @@ def prepare_reference_b0(in_dwi, in_bval, in_bvec, low_bval=5, working_directory
     )
 
     return out_reference_b0, out_b0_dwi_merge, out_updated_bval, out_updated_bvec
+
+
+def delete_apply_transform(marker, dir_to_del, base_dir, light):
+    import shutil
+    from pathlib import Path
+
+    print("\n", marker, "\n")
+    print("\n", base_dir, "\n")
+    print("\n", dir_to_del, "\n")
+    if not light:
```

Comment:
I don't understand why you need `light`. You can simply add the cleaning node to the pipeline conditionally to the value of the boolean parameter (called `light_version` for now), or am I missing something?